src/Components/MainComponent/Restaurants/index.js

```diff
@@ -1,18 +1,20 @@
 import Restaurant from "./Restaurant";
 
-const Restaurants = ({restaurantItems}) => {
+const Restaurants = ({restaurantItems, setRestaurantID, showingRestaurants}) => {
 
     return (
-        <div className="row px-3">
+        <div className={showingRestaurants}>
```

Comment:
This div is inside the preceding one, hence needs indenting. This will affect the indenting of following lines.